### ISSUE TITLE
switch to eddy_cuda10.2

### ISSUE THREAD

**Issue by mattcieslak** [octomike/eddy_cuda_findings#1]
Hi Mike,

I'm thinking about switching the default eddy_cuda to eddy_cuda10.2 and using nvidia/cuda:10.2-runtime-ubuntu18.04 as the base image. Would that mess up your workflow?

**Comment by octomike** [comment]
No, not at all. We are actually using a temporary container with exactly those changes. :) see  my PR here https://github.com/PennLINC/qsiprep/pull/334
